src/ecs-mcp-server/awslabs/ecs_mcp_server/modules/containerize.py

```diff
@@ -28,7 +29,15 @@
 def register_module(mcp: FastMCP) -> None:
     """Register containerize module tools and prompts with the MCP server."""
 
-    @mcp.tool(name="containerize_app", annotations=None)
+    @mcp.tool(
```

Comment:
LGTM - containerize is read-only and only provides guidance. Guidance is hard-coded.